### PR TITLE
Updated Prometheus Invocation

### PR DESCRIPTION
**Pull request by nils-**
Updating documentation: Prometheus Invocation to include config file.

### CODE REVIEW

File: doc/prometheus/README.md [hunk 1]
```diff
@@ -15,6 +15,6 @@ It is recommended that 5 seconds be the minimum that the scrape interval is set
 Invocation
 ----
 
-    java -jar prometheus.jar prometheus_port=9001 prometheus_path=/metrics
+    java -jar prometheus.jar prometheus_port=9001 prometheus_path=/metrics config=/config.cfg
```
Comment (dmjb) [inline doc/prometheus/README.md]: Is this a URL endpoint, or a file path?

Comment (nahratzah) [inline doc/prometheus/README.md]: Absolute file path.

Can you change it to ``/path/to/monsoon/config.cfg`` or something along those lines?  I don't think monsoon is so important that we should put its config in the root of the filesystem. ;)

Comment (dmjb) [inline doc/prometheus/README.md]: What @nahratzah said ;)

**Comment by nahratzah** [comment]
LGTM

File: doc/prometheus/README.md [hunk 2]
```diff
@@ -15,6 +15,6 @@ It is recommended that 5 seconds be the minimum that the scrape interval is set
 Invocation
 ----
 
-    java -jar prometheus.jar prometheus_port=9001 prometheus_path=/metrics
+    java -jar prometheus.jar prometheus_port=9001 prometheus_path=/metrics config=/path/to/monsoon/config.cfg
```
Comment (dmjb) [inline doc/prometheus/README.md]: Similarly, is this promethus path a file path or a URL?

**Comment by nils-** [comment]
This is the endpoint that the Prometheus module exposes the metrics on.

**Comment by dmjb** [comment]
:shipit:

**Pull request merged**